CONTRIBUTING.md

```diff
@@ -87,6 +88,36 @@ Our Travis script will run jshint over all the source.  You should run jshint as
 Many editors and IDEs do this automatically for you -- Atom, PyCharm, TextMate has a
 nice plugin called JSLintMate.  You can easily install [jshint](http://jshint.org) using the ``npm`` package manager that comes with [node](http://nodejs.org).
 
+
+## Committers
+
+The committers are people who are responsible for reviewing and approving pull requests .  These are 
+generally people who have been around the project for a while and have "proven" themselves by contributing
+good code and ideas to the Skulpt.  This list may change over time as people gain or lose interest in
+Skulpt.  If you would like to volunteer contact Brad.
+
+Although a lot of our testing, and checking for adherence to the style guidelines is done automatically
+the review process is generally as follows:
+
+1.  Look at the diffs for each file on github, if it is pretty obvious what they are doing then great.
+2.  Look at the tests provided with the PR and try to think if there are additional tests that would provide better coverage. If you think additional tests are needed
+then you should let the owner of the PR know and have them add more tests.  I think that all new PRs that are adding new features
```

Comment:
The 'I think that' can probably be removed for clarity.  Seems these guidelines are most helpful if they're opinionated about what's good vs bad.